darwin/dataset/download_manager.py

```diff
@@ -82,7 +82,7 @@ def download_all_images_from_annotations(
         raise ValueError(f"Annotation format {annotation_format} not supported")
 
     # Verify that there is not already image in the images folder
-    unfiltered_files = images_path.rglob("*") if use_folders else images_path.glob("*")
+    unfiltered_files = images_path.rglob("*") if use_folders else images_path.rglob("*")
```

Comment:
Users may download flat releases of a dataset **after** downloading releases with the folder structure